test/tictactoe/game_menu_test.clj

```diff
@@ -0,0 +1,54 @@
+(ns tictactoe.game-menu-test
+  (:require [clojure.test :refer :all]
```

Comment:
`:refer :all` is a bad practice in most cases. I basically *always* prefer `:as`.

`:as` > `:refer [a b c]` > `:refer :all` > `:use`